python/xorbits/web/ui/src/node_info/NodeResourceTab.js

```diff
@@ -61,25 +61,74 @@ export default class NodeResourceTab extends React.Component {
 
   generateBandRows() {
     const converted = {}
+
+    const tablecontentReturn = (converted, band) => {
+      const table_style = (
+        <TableRow>
+          <TableCell rowSpan={3}>{band}</TableCell>
+        </TableRow>
+      )
+      const table_content = Object.keys(converted[band]).map((item) => (
+        <TableRow key={item}>
+          <TableCell align="right">{item}</TableCell>
+          {Object.keys(converted[band][item]).map((k) => (
+            <TableCell key={k} align="right">
+              {converted[band][item][k]}
+            </TableCell>
+          ))}
+        </TableRow>
+      ))
+
+      let table_total = [table_style, table_content]
+      return table_total
+    }
+
     Object.keys(this.state.resource).map((band) => {
       let detail = this.state.resource[band]
-      converted[band] = {
-        CPU: {},
-        Memory: {},
-      }
-      let cpuAvail = detail['cpu_avail']
-      let cpuTotal = detail['cpu_total']
-      let cpuUsage = (cpuTotal - cpuAvail).toFixed(2)
+      console.log(detail)
+      console.log(band)
+      if (band === 'numa-0') {
+        converted[band] = {
+          CPU: {},
+          Memory: {},
+        }
+
+        let cpuAvail = detail['cpu_avail']
+        let cpuTotal = detail['cpu_total']
+
+        console.log(cpuAvail)
+        let cpuUsage = (cpuTotal - cpuAvail).toFixed(2)
+
+        let memAvail = detail['memory_avail']
+        let memTotal = detail['memory_total']
+        let memUsage = toReadableSize(memTotal - memAvail)
+
+        converted[band]['CPU']['CpuUsage'] = cpuUsage
+        converted[band]['CPU']['CpuTotal'] = cpuTotal.toFixed(2)
 
-      let memAvail = detail['memory_avail']
-      let memTotal = detail['memory_total']
-      let memUsage = toReadableSize(memTotal - memAvail)
+        converted[band]['Memory']['MemUsage'] = memUsage
+        converted[band]['Memory']['MemTotal'] = toReadableSize(memTotal)
+      } else if (band === 'gpu-0') {
```

Comment:
Why always ``gpu-0``? If there are multiple gpus on the machine, what's the situation?

Reply:
try to fix in the latest version, pending verification.

Reply:
resolved in the latest edition, see the most recent comment below for details.